consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/IbftProcessor.java

```diff
@@ -48,6 +50,10 @@ public void stop() {
     shutdown = true;
   }
 
+  public void awaitStop() throws InterruptedException {
```

Comment:
Not sure I understand the question.  The IBFT coordinator will try to wait for everything to shut down, but there's an absolute timeout to avoid us hanging forever.

Reply:
We can't wait forever.  This won't be an issue unless shutdown is already going poorly.